language-plutus-core/generators/Language/PlutusCore/Generators/Test.hs

```diff
@@ -79,7 +79,7 @@ sampleProgramValueGolden folder name genTerm = do
 -- Checks whether a term generated along with the value it's supposed to compute to
 -- indeed computes to that value according to the provided evaluate.
 propEvaluate
-    :: (uni ~ DefaultUni, Pretty internal, KnownType uni a)
+    :: (uni ~ DefaultUni, Pretty internal, KnownType uni a, uni `Everywhere` PrettyConst)
```

Comment:
The new constraint should be redundant. We already know that ``DefaultUni `Everywhere` PrettyConst``.

Reply:
Well spotted.  I think I added that when I was trying to fix the EvaluationResult thing.